packages/react/src/components/VideoPlayer/VideoPlayer.js

```diff
@@ -55,7 +58,12 @@ const VideoPlayer = ({ inverse, showCaption, videoId, customClassName }) => {
       </div>
       {showCaption && (
         <div className={`${prefix}--video-player__video-caption`}>
-          {videoData.name} {videoDuration}
+          {videoData.description &&
+            markdownToHtml(videoData.description, {
+              textOnly: true,
```

Comment:
@annawen1 you will need to update this to `createParagraphs: false`

Reply:
ack right